app/androidutils.h

```diff
@@ -10,10 +10,9 @@
 #ifndef ANDROIDUTILS_H
 #define ANDROIDUTILS_H
 
-
 #ifdef ANDROID
-#include <QAndroidActivityResultReceiver>
-#include <QAndroidJniObject>
+#include <QtCore/private/qandroidextras_p.h>
```

Comment:
uff, why this is needed? I thought androidextras are dead?

Reply:
all of those classes should be part of QtCore now no?

Reply:
According to https://www.qt.io/blog/qt-extras-modules-in-qt-6 and https://doc-snapshots.qt.io/qt6-6.3/qtandroidprivate.html no, they are not available in the QtCore. There are only few things in the https://doc-snapshots.qt.io/qt6-6.3/qnativeinterface-qandroidapplication.html